layouts/default.vue

```diff
@@ -270,4 +392,79 @@
 		background: var(--dark-500);
 		border-color: var(--dark-400);
 	}
+
+	.n-header-subj {
+		@apply dark:!text-dark-0;
+		color: #4b5563;
+	}
+
+	.btn-mark-all-as-read {
+		@apply flex h-10 items-center gap-x-2;
+		@apply bg-white;
+		@apply text-bluegray-900;
+		@apply h-10 w-full sm:w-auto;
+		@apply border border-solid !border-[var(--p-surface-300)];
+		@apply hover:!border-[var(--p-primary-500)] hover:!bg-[var(--p-primary-500)] hover:!text-[var(--bluegray-0)];
+		@apply dark:text-[var(--bluegray-0)] dark:bg-primary dark:!border-primary;
+		@apply dark:hover:!bg-[var(--p-primary-hover-color)] dark:hover:!border-[var(--p-primary-hover-color)];
+	}
+
+	.n-accordion {
+		@apply box-border flex w-full flex-col gap-y-2;
+	}
+
+	.n-accordion-panel {
+		@apply border-none dark:border dark:border-solid dark:border-[var(--table-border)];
+		@apply bg-[var(--p-surface-50)] dark:bg-dark-800;
+		@apply !p-0 !pb-4;
+	}
+
+	.n-accordion-panel[data-pc-name="accordionpanel"] {
+		@apply !rounded-xl;
+	}
+
+	.n-accordion-panel_new {
+		@apply bg-gradient-to-r from-[rgba(244,252,247,1)] to-[rgba(229,252,246,1)];
+		@apply dark:bg-none dark:!bg-[var(--dark-700)];
+	}
+
+	.n-accordion-panel_new .n-header-subj {
+		@apply !text-[var(--bluegray-900)];
+		@apply dark:!text-[var(--bluegray-0)];
```

Comment:
Base fixes of text colors and backgrounds, can be applied directly at preset files. So for something specific => local overrides, for changes like replacing `text-surface-800` with `text-bluegray-900` => global theme override.

Reply:
@MartinKolarik since there is https://github.com/jsdelivr/globalping-dash/pull/61#discussion_r1950831769 could you please confirm that is ok? In 90% of the cases it is only fixing colors for dark theme (e.g. https://github.com/jsdelivr/globalping-dash/pull/59/files#diff-6fd410bbd76536ad0916987fb9292b0fcc746a5b7becc71554e20988d30979bc)

Reply:
Sure, fixing theme colors most likely applies to all occurrences of the component so preset.